content/BPIP-3.md

```diff
@@ -1,7 +1,8 @@
 ---
 bpip: 3
 title: Pre-minted Vouchers (Shell rNFTs)
-discussions-to: 
+author: Cliff Hall
+discussions-to: https://github.com/bosonprotocol/BPIPs/discussions
```

Comment:
This would be a better link: https://github.com/bosonprotocol/BPIPs/issues/3

Reply:
@thecryptofruit updated